test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -721,9 +721,34 @@ context('action sidebar', function() {
       },
     });
 
+    const testFiles = [
+      {
+        id: uuid(),
+        attributes: {
+          path: `patients/${ testPatient.id }/HRA.pdf`,
+          created_at: '2019-08-24T14:15:22Z',
+        },
+        meta: {
+          view: `https://www.bucket_name.s3.amazonaws.com/patients/${ testPatient.id }/view/HRA.pdf`,
+          download: `https://www.bucket_name.s3.amazonaws.com/patients/${ testPatient.id }/download/HRA.pdf`,
+        },
+      },
+      {
+        id: uuid(),
+        attributes: {
+          path: `patients/${ testPatient.id }/HRA v2.pdf`,
+          created_at: '2019-08-25T14:15:22Z',
+        },
+        meta: {
+          view: `https://www.bucket_name.s3.amazonaws.com/patients/${ testPatient.id }/view/HRA%20v2.pdf`,
+          download: `https://www.bucket_name.s3.amazonaws.com/patients/${ testPatient.id }/download/HRA%20v2.pdf`,
+        },
+      },
+    ];
+
     const testAction = getAction({
       relationships: {
-        'files': getRelationship([{ id: '1' }, { id: '2' }], 'files'),
+        'files': getRelationship([{ id: testFiles[0].id }, { id: testFiles[1].id }], 'files'),
```

Comment:
if this was a getFile it would return the type and then this could just be `getRelationship(testFiles),`

Reply:
Ya that would clean things up in a number of places 👍